src/command_panel.nim

```diff
@@ -0,0 +1,38 @@
+import strutils
+import typetraits
+
+proc commandPanel*(): tuple[y: int, x: int] =
```

Comment:
@stephenc222 
rename your proc to `renderCommandPanel` to avoid the import problem I mentioned on slack.